lib/protoparser/zabbixconnector/parser.go

```diff
@@ -0,0 +1,267 @@
+package zabbixconnector
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/VictoriaMetrics/VictoriaMetrics/lib/logger"
+	"github.com/VictoriaMetrics/metrics"
+	"github.com/valyala/fastjson"
+)
+
+const (
+	ADDGROUPS    byte = 0b00000001
+	ADDEMPTYTAGS byte = 0b00000010
+	EMPTYVALUE   byte = 49 // "1"
```

Comment:
Could you elaborate on why `1` is used as empty value? Is this something specific for zabbix? 
If not, wdyt about using something like `[empty]` and making this configurable by user?

Also, constants in Go are named in `CamelCase` where the first letter will make constant exported or private. As far as I can see, `emptyValue` can be used here since constant is only used by this package so it can be unexported.

Reply:
Agree. Now command-line flags set the values for labels. If no flags are specified, groups and empty tags are ignored.
`-zabbixconnector.addGroups=available`
`-zabbixconnector.addEmptyTags=empty`